requirements.txt

```diff
@@ -37,7 +37,6 @@ docutils==0.13.1
 easy-thumbnails==2.3
 elasticsearch==2.4.1
 elasticsearch-dsl==2.2.0
-git+https://github.com/CSIS-iLab/fieldbook-python.git@8b153e#fieldbook
```

Comment:
I think this was the only thing that required `requests`. It was also the only module which was using `nameparser`. Both of those requirements can likely be removed.